docs/FAQ.md

```diff
@@ -49,10 +49,14 @@ use `jj squash` to amend the previous commit.
 
 For more options see the next question.
 
-### Can I add a portion of the edits I made to a file, similarly to `git add -p` or `hg commit -i`?
+### Can I create a new commit from only some of the changes in the working copy, like `git add -p && git commit` or `hg commit -i`?
```

Comment:
This should probably clarify "interactively" (see https://github.com/martinvonz/jj/issues/2393#issuecomment-1766957167).